nucleus/dataset.py

```diff
@@ -102,7 +102,7 @@ def annotate(
         self,
         annotations: List[Union[BoxAnnotation, PolygonAnnotation]],
         update: Optional[bool] = DEFAULT_ANNOTATION_UPDATE_MODE,
-        batch_size: int = 20,
+        batch_size: int = 5000,
```

Comment:
Intentional?

Reply:
intentional, the batch_size is way too small.